src/api/helpers/map-CIS-countries/map-CIS-country/no-insurance-support/index.test.ts

```diff
@@ -0,0 +1,116 @@
+import noInsuranceSupport from '.';
+import { EXTERNAL_API_DEFINITIONS } from '../../../../constants';
+
+const {
+  CIS: {
+    COUNTRY_RATINGS,
+    ESRA_CLASSIFICATION: { VERY_HIGH, HIGH, STANDARD, NONE },
+    SHORT_TERM_COVER: { UNLISTED, CILC },
+  },
+} = EXTERNAL_API_DEFINITIONS;
+
+const mockInitCountry = {
+  esraClassification: NONE,
+  shortTermCover: UNLISTED,
+};
+
+const createMockParams = (countryRating: string, esraClassification?: string, shortTermCover?: string) => {
```

Comment:
could this go in test-mocks?

Reply:
I think that'd be overkill because it's very specific to this unit test and won't be used anywhere else.